tests/test_raocp.py

```diff
@@ -0,0 +1,238 @@
+import unittest
+import raocp.core as rc
+import numpy as np
+import raocp.core.cones as core_cones
+
+
+class TestRAOCP(unittest.TestCase):
```

Comment:
Unit tests must be **unit** tests and test suites should test things in a single class or package. Read [this](https://www.toptal.com/qa/how-to-write-testable-code-and-why-it-matters).